youtube_dl/extractor/blerp.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from .common import InfoExtractor
+
+
+class BlerpIE(InfoExtractor):
+    IE_NAME = 'blerp'
+    _VALID_URL = r'https?://(?:www\.)?blerp\.com/soundbites/(?P<id>[0-9a-zA-Z]+)'
+    _TESTS = [{
+        'url': 'https://blerp.com/soundbites/6320fe8745636cb4dd677a5a',
+        'info_dict': {
+            'id': '6320fe8745636cb4dd677a5a',
+            'title': 'Samsung Galaxy S8 Over the Horizon Ringtone 2016',
+            'uploader': 'luminousaj',
+            'uploader_id': '5fb81e51aa66ae000c395478',
+            'ext': 'mp3',
+            'keywords': 'samsung,galaxy,s8,over the horizon,2016,ringtone',
```

Comment:
Extract these as a list named `tags`:
```suggestion
            'tags': ['samsung', 'galaxy', 's8', 'over the horizon', '2016', 'ringtone'],
```